rir/src/interpreter/interp.cpp

```diff
@@ -91,7 +91,7 @@ void endClosureContext(RCNTXT* cntxt, SEXP result) {
 }
 
 INLINE SEXP createPromise(Code* code, SEXP env) {
-    SEXP p = mkPROMISE((SEXP)code, env);
+    SEXP p = Rf_mkPROMISE((SEXP)code, env);
```

Comment:
What's the reason for this change? Nothing wrong with it -- I'm just not familiar with the code base and conventions.

Reply:
My reasoning was that this makes it more clear that we are calling into R. `Rf_` is short for `R function`, I think?
So this way: `createPromise(...)` -> calling sth. from RIR, vs. `Rf_mkPROMISE(...)` -> calling R

Reply:
Makes sense, thanks!